mlflow/models/__init__.py

```diff
@@ -28,13 +28,13 @@ def to_yaml(self, stream=None):
         return yaml.safe_dump(self.__dict__, stream=stream, default_flow_style=False)
 
     def save(self, path):
-        """Write this Servable as a YAML file to a local file."""
+        """Write this servable as a YAML file to a local file."""
```

Comment:
We should write "model" instead of "servable" everywhere here. Servable was an older name for it.